std/zig/parser_test.zig

```diff
@@ -470,6 +470,34 @@ test "zig fmt: array literal with hint" {
     );
 }
 
+test "zig fmt: array literal veritical column alignment" {
+    try testTransform(
+        \\const a = []u8{
+        \\    1000, 200,
+        \\    30, 4,
+        \\    50000, 60
+        \\};
+        \\const a = []u8{0,   1, 2, 3, 40,
+        \\    4,5,600,7,
+        \\           80,
+        \\    9, 10, 11, 0, 13, 14, 15};
+        \\
+        ,
+        \\const a = []u8{
+        \\    1000,  200,
+        \\    30,    4,
+        \\    50000, 60,
+        \\};
+        \\const a = []u8{
+        \\    0,  1,  2,   3, 40,
```

Comment:
Do we want items to be left-justified, is right-justified a better option?

Reply:
The [accepted proposal](https://github.com/ziglang/zig/issues/1793) demonstrates left-justified columns in the description, which is why I implemented that here. I personally happen to prefer it that way -- and it might start to look funny with array elements which aren't numbers (strings, other arrays).

Reply:
It is possible to handle numbers different than other types - the render has access to know what kind of ast nodes and tokens something is, so it could detect if there were strings, numbers, etc in the array. If we want to go that route though, let's open a new proposal for it.